docs/interpreter/springxd.md

```diff
@@ -0,0 +1,150 @@
+SpringXD supports two processing abstractions: (1) [Streams](http://docs.spring.io/spring-xd/docs/current/reference/html/#streams) for event driven data, and (2) [Jobs](http://docs.spring.io/spring-xd/docs/current/reference/html/#jobs) like MR, PIG, Hive, Cascading, SQL and so on, for batch data types. Spring XD tries to blur the boundaries between the two using a channel abstraction, so that, for example, a stream can trigger a batch job, and a batch job can send events and, in turn, trigger other streams. An expressive DSL is provided for defining streams and jobs: 
+
+```
+myStream = http | filter --expression=payload='foo' | log
+```
+
+Zeppelin provides interpreters for SpringXD `Streams` and `Jobs`: 
+
+<table class="table-configuration">
+  <tr>
+    <th>Name</th>
+    <th>Class</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>%xd.stream</td>
+    <td>SpringXdStreamInterpreter</td>
+    <td>Provides an environment to create SpringXD Streams - defines the ingestion of event driven data from a `source` to a sink that passes through any number of `processors. </td>
+  </tr>
+  <tr>
+    <td>%xd.job</td>
+    <td>SpringXdJobInterpreter</td>
+    <td>Provides an environment to create SpringXD Jobs - combine interactions with standard enterprise systems (e.g. RDBMS) as well as Hadoop operations (e.g. MapReduce, HDFS, Pig, Hive or HBase). </td>
+  </tr>  
+</table>
+
+> Note: A single Zeppelin `paragraph` can contain multiple `streams` or multiple `jobs`, but it can not mix-up jobs and streams in the same paragraph!
+
+<br/>
+[<img align="right" src="http://img.youtube.com/vi/uSUBe8Hgk8w/0.jpg" alt="zeppelin-view" hspace="10" width="250"></img>](https://www.youtube.com/watch?v=uSUBe8Hgk8w)
+
+Following [video tutorial](https://www.youtube.com/watch?v=uSUBe8Hgk8w) illustrates some of the features provided by the `SpringXD Interpreter`. It shows how to build SpringXD pipeline to ingest a real-time Tweet stream into Geode, HDFS and HAWQ backends. 
+
+### Create Interpreter
+
+By default Zeppelin creates one `SpringXD` interpreter instance that communicates with the backend SpringXD system
+
+To create new SpringXD instance open the `Interpreter` section and click the `+Create` button. Pick a `Name` of your choice and from the `Interpreter` drop-down select `xd`. Then follow the configuration instructions and `Save` the new instance.
+
+> Note: The `Name` of the instance is used only to distinct the instances while binding them to the `Notebook`. The `Name` is irrelevant inside the `Notebook`. In the `Notebook` you must use `%xd.stream` or `%xd.job` tag.
+
+### Bind to Notebook
+In the `Notebook` click on the `settings` icon in the top right corner. The select/deselect the XD interpreter to be bound with the `Notebook`.
```

Comment:
Maybe, I think "The select/deselect the ~" -> "Then select/deselect the ~" is correct.